ghost/i18n/locales/es/portal.json

```diff
@@ -6,171 +6,153 @@
-    "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "Añade {{senderEmail}} a tu lista de contactos. Esto le indica a tu provedor de correo electrónico que puede confiar en esta dirección de correo.",
-    "Invalid email address": "",
-    "Jamie Larson": "",
-    "jamie@example.com": "",
+    "If you have a corporate or government email account, reach out to your IT department and ask them to allow emails to be received from {{senderEmail}}": "Si tienes un correo electrónico corporativo o del gobierno, puedes comunicarte con el departamento de TI y preguntarles que permitan recibas correos de {{senderEmail}}",
+    "If you would like to start receiving emails again, the best next steps are to check your email address on file for any issues and then click resubscribe on the previous screen.": "Si deseas comenzar a recibir correos electrónicos nuevamente, lo mejor es verificar tu dirección de correo electrónico registrada para detectar cualquier problema y luego resuscribirse en la pantalla anterior.",
+    "If you're not receiving the email newsletter you've subscribed to, here are a few things to check.": "Si no estás recibiendo los boletines informativos a los que te suscribiste, estas son algunas cosas que puedes hacer.",
+    "If you've completed all these checks and you're still not receiving emails, you can reach out to get support by contacting {{supportAddress}}.": "Si haz comprobado todo y aun no estás recibiendo los correos electrónicos, comunicarte con soporte técnico contactando a {{supportAddress}}.",
+    "In the event a permanent failure is received when attempting to send a newsletter, emails will be disabled on the account.": "En el caso de que haya un problema permanente al intentar enviar un boletín, los correos electrónicos serán desabilitados en su cuenta.",
+    "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "Añade este correo electrónico {{senderEmail}} a tu lista de conactos confiados. Esto le indica a tu provedor de correo electrónico, que puedes recibir correos de esta dirección de correo sin problemas.",
     "Less like this": "Menos como esto",
     "Make sure emails aren't accidentally ending up in the Spam or Promotions folders of your inbox. If they are, click on \"Mark as not spam\" and/or \"Move to inbox\".": "Asegúrate de que los correos electrónicos no terminen accidentalmente en las carpetas de correo no deseado o promociones de su bandeja de entrada. Si lo son, haz clic en \"Marcar como no spam\" y/o \"Mover a la bandeja de entrada\".",
     "Manage": "Administrar",
     "Maybe later": "Tal vez más tarde",
     "Memberships unavailable, contact the owner for access.": "Membresía no disponible, contacta al propietario para obtener acceso.",
-    "month": "",
```

Comment:
please do not delete - this string is needed to show the monthly price. (i.e. $5/month in English.)

Reply:
resolved.